silverback/types.py

```diff
@@ -1,9 +1,20 @@
+from enum import Enum
 from typing import Optional, Protocol
 
 from pydantic import BaseModel
 from typing_extensions import Self  # Introduced 3.11
 
 
+class TaskType(str, Enum):
+    STARTUP = "silverback_startup"  # TODO: Shorten
```

Comment:
If these are wanted breaking changes, I like to put the version in the TODO so I can go back and look them all for like a misc-breaking-changes.

```
TODO: Shorten in 0.4.0
```

Reply:
This is more internal API (the middleware uses this value), but probably best to do it as a breaking update

Reply:
[c177a94](https://github.com/ApeWorX/silverback/pull/66/commits/c177a94e658af23e48c66f114e06578000da8241)